scripts/error-codes/codes.json

```diff
@@ -416,5 +416,6 @@
   "428": "useServerContext is only supported while rendering.",
   "429": "ServerContext: %s already defined",
   "430": "ServerContext can only have a value prop and children. Found: %s",
-  "431": "React elements are not allowed in ServerContext"
+  "431": "React elements are not allowed in ServerContext",
+  "432": "Unsupported server component type: %s.%s"
```

Comment:
Should I just update the existing code (351), or is it necessary to append a new entry?